lib/fork.js

```diff
@@ -19,8 +19,16 @@ module.exports = function (args) {
 	var ps = childProcess.fork(babel, args, options);
 
 	var promise = new Promise(function (resolve, reject) {
+		// define backup results, when there are no tests defined
+		var testResults = {
+			stats: {
+				passCount: 0
```

Comment:
I would agree, it needs a separate issue/discussion.

I think your code is fine, no need to revert. Lets just create a new issue and follow up on it.